File: SistemaProcinco/SistemaProcinco.Common/Models/ParticipanteViewModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SistemaProcinco.Common.Models
{
    public class ParticipanteViewModel
    {
        public int Part_Id { get; set; }
        public string Part_DNI { get; set; }
        public int? Empre_Id { get; set; }
        public string Part_Nombre { get; set; }
        public string Part_Apellido { get; set; }
        public string Part_Correo { get; set; }
        public DateTime? Part_FechaNacimiento { get; set; }
        public string Part_Sexo { get; set; }
        public int? Estc_Id { get; set; }
        public string Part_Direccion { get; set; }
        public string Ciud_Id { get; set; }
        public int? Part_UsuarioCreacion { get; set; }
        public DateTime? Part_FechaCreacion { get; set; }
        public int? Part_UsuarioModificacion { get; set; }
        public DateTime? Part_FechaModificacion { get; set; }
        public bool? Part_Estado { get; set; }





        [NotMapped]
        public string UsuarioCreacion { get; set; }

        [NotMapped]
        public string UsuarioModificacion { get; set; }

        [NotMapped]
        public string Esta_Id { get; set; }
        [NotMapped]
        public string Ciud_Descripcion { get; set; }
        [NotMapped]
        public string Sexo { get; set; }

        [NotMapped]
        public string EstadoCivil { get; set; }

        [NotMapped]
        public string Empre_Descripcion { get; set; }
    }
}

File: PlantillaProcinco/src/app/Models/ParticipanteViewModel.ts
export class Participante {
    part_Id !: Number;
    part_DNI  !: String;

    empre_Id !: Number;
    empre_Descripcion !: String;
    part_Nombre !: String;
    part_Apellido !: String;
    part_Correo !: String;
    part_FechaNacimiento !: String;
    part_Sexo !: String;
    sexo !: String;
    estc_Id !: Number;
    estadoCivil !: String;
    part_Direccion !: String;
    ciud_Id !: String;
    ciud_Descripcion !: String;
    empl_UsuarioCreacion !: Number;
    empl_FechaCreacion !: String;
    empl_UsuarioModificacion !: Number;
    empl_FechaModificacion !: String;
    usuarioCreacion!: String;
    usuarioModificacion!: String;
    esta_Id!: String;
}






export class Cursoo{
    curso_Id !: Number;
    curso_Descripcion  !: String;
    curso_DuracionHoras  !: String;
    curso_Imagen  !: String;
    cate_Id  !: String;
    empre_Id  !: String;
    categoria  !: String;
}

export class CursoImpartidoEnviar {
    Curso_Id?:String;
    txtCurso: string;
    txtxEmpresa: string;
    txtxHoras: string;
    txtImagen: string;
    txtCategoria: string;
    contenidosSeleccionados: any;
}



export class Fill{
    curso_Id: string;
    part_Id: string;
    curso_Descripcion  !: String;
    curso_DuracionHoras  !: String;
    curso_Imagen  !: String;
    cate_Id  !: String;
    empre_Id  !: String;
    categoria  !: String;
    usuarioCreacion: string;
    usuarioModificacion: string;
    fechaCreacion : string;
    fechaModificacion : string;
}